astro/src/css/style.css

```diff
@@ -8,7 +8,7 @@
 
 lite-youtube{
   @apply aspect-video;
-  max-width: none;
+  max-width: none !important;
```

Comment:
why is this needed?

Reply:
https://inversoft.slack.com/archives/C02CR2M6SRK/p1697738837291899

Reply:
Alright the tailwind class for this is `max-w-full` which would be preferable to using `!important`